translations/ui-courses/en.json

```diff
@@ -167,5 +169,7 @@
     "lookUpUser": "Look up user",
     "delete": "Delete",
     "reallyDelete": "Really delete?",
+    "addFastAddItem.title": "Add Fast Add item",
```

Comment:
Can't we find less clumsy wording than "Add Fast Add item"? (The UX otherwise looks find to my non-specialist eyes.)

Reply:
Just to have everything in one place ... when I stupidly asked this question as a comment of its own (outside of this review), @doytch replied:

> Honestly I'm all ears for alternatives. I don't love it either but couldn't come up with anything better that didn't start destroying consistency. :\ My hope is that when this gets put in front of the SIG they have better ideas, but the fact that this sort of functionality has been called "Fast Add" in Folio constrains the option space a bit.

That seems fine to me, then — let's wait and see what the SIG says.